lib/rester/errors.rb

```diff
@@ -16,8 +16,10 @@ class MethodDefinitionError < Error; end
 
     #############
     # Stub Adapter Errors
-    class StubAdapterError < Error; end
-    class InvalidStubFileError < StubAdapterError; end
+    class StubError < Error; end
+
+    #############
+    # Stub Adapter Errors
```

Comment:
Remove duplicate of "Stub Adapter Errors" comment heading.

Also, rename the first one to "Stub Errors"